docs/0.5/tutorials/configuring_splinter_nodes_with_docker.md

```diff
@@ -242,30 +264,32 @@ Congratulations! You've got a Splinter node up and running.
         volumes:
           - beta-var:/var/lib/splinter
           - ./config:/config
-          - allow_keys_beta:/etc/splinter/allow_keys
+          - ./allow_keys_beta:/etc/splinter/allow_keys
         entrypoint: |
           bash -c "
             splinter cert generate --skip && \
+            splinter keygen --system --skip && \
+            splinter database migrate && \
             splinterd -v \
                 --node-id beta \
                 --network-endpoints tcps://0.0.0.0:8044 \
                 --advertised-endpoints tcps://splinterd-beta:8044 \
-                --rest-api-endpoint 0.0.0.0:8080 \
+                --rest-api-endpoint http://0.0.0.0:8080 \
                 --registries file:///config/beta-registry.yaml \
-                --storage yaml \
                 --tls-insecure
           "
 
     ```
 
-1. Create the circuit admin keys for Beta.
+1. Create the circuit admin keys for Beta and get the daemon.
```

Comment:
Should the last half of this sentence be get the daemon key? Or perhaps "start" instead of get?

Reply:
removed